components/header/DarkModeToggle.tsx

```diff
@@ -0,0 +1,25 @@
+import { Box, useColorMode } from "@chakra-ui/react";
+
+import { colors } from "../../theme/colors";
+
+const DarkModeToggle = () => {
```

Comment:
Please change to ColorModeToggle, it could be done in two ways Light -> Dark and opposite.